src/main/java/seedu/hrpro/model/ModelManager.java

```diff
@@ -241,13 +243,18 @@ public void addStaffToProject(Index projectIndex, Staff staff) {
     }
 
     @Override
-    public void editStaffInProject(ProjectName projectName, Staff toEdit, Staff editWith) {
+    public boolean isSuccessStaffEdit(ProjectName projectName, Staff toEdit, Staff editWith) {
```

Comment:
Nice catch. I think the naming could be worked on, since `isSuccessStaffEdit` doesn't sound like an edit command method. Perhaps changing to `staffEdit` that returns a `boolean`?